src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1530,5 +1530,19 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+
+		// PlayerEnterChunkEvent
+		EventValues.registerEventValue(PlayerMoveEvent.class, Chunk.class, new Getter<Chunk, PlayerMoveEvent>() {
+			@Override
+			public Chunk get(PlayerMoveEvent event) {
+				return event.getFrom().getChunk();
+			}
+		}, EventValues.TIME_PAST);
+		EventValues.registerEventValue(PlayerMoveEvent.class, Chunk.class, new Getter<Chunk, PlayerMoveEvent>() {
+			@Override
+			public Chunk get(PlayerMoveEvent event) {
+				return event.getTo().getChunk();
+			}
+		}, EventValues.TIME_NOW);
```

Comment:
Instead of these event values, you should register a location to chunk converter. (Surprised one doesn't exist already)

This will allow event-chunk by using the already registered location event values.

Reply:
Ooo that sounds amazing! Will do so!